eclipse-theia/theia
# Welcome Widget: Mark recent workspaces as DevContainer

#### What it does

In Recent Workspaces List in Welcome Widget workspaces which were recently opened in a DevContainer are marked as such.
 
#### How to test

Open a workspace in DevContainer. Reopen the Theia app and see the last workspace marked as DevContainer.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
